integrations/langchain/src/databricks_langchain/chat_models.py

```diff
@@ -557,68 +557,78 @@ def _stream(
 
         data = self._prepare_inputs(messages, stop, stream=True, **kwargs)
 
+        usage_chunk_emitted = False
+        final_usage = None
+
         if self.use_responses_api:
-            # Use OpenAI client with responses API for streaming
             prev_chunk = None
             stream = self.client.responses.create(**data)  # type: ignore
             for chunk in stream:
                 chunk_message = _convert_responses_api_chunk_to_lc_chunk(chunk, prev_chunk)
                 prev_chunk = chunk
                 if chunk_message:
                     yield ChatGenerationChunk(message=chunk_message)
+                # Check for usage in the chunk if available
+                if stream_usage and hasattr(chunk, "usage") and chunk.usage:
+                    input_tokens = getattr(chunk.usage, "prompt_tokens", None)
+                    output_tokens = getattr(chunk.usage, "completion_tokens", None)
+                    if input_tokens is not None and output_tokens is not None:
+                        final_usage = {
+                            "input_tokens": input_tokens,
+                            "output_tokens": output_tokens,
+                            "total_tokens": input_tokens + output_tokens,
+                        }
+            # Emit special usage chunk at end of stream
```

Comment:
Thanks for this fix! Curious why emit the usage chunk only at the end of the stream, instead of with each chunk? Do we know how this behavior compares to e.g. ChatOpenAI from langchain?